public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/route-settings/RouteSettings.tsx

```diff
@@ -74,15 +78,6 @@ export const RoutingSettings = ({ alertManager }: RoutingSettingsProps) => {
                 if (!value || value.length === 0) {
                   return 'At least one group by option is required.';
                 }
-                // if value includes only the allowedSingleFieldInGroupBy, it is valid
```

Comment:
there is no need to validate this now, as these options in select are fixed and not removable